openedx/core/apidocs.py

```diff
@@ -10,7 +11,7 @@
     version="v1",
     description="APIs for access to Open edX information",
     #terms_of_service="https://www.google.com/policies/terms/",         # TODO: Do we have these?
-    email="oscm@edx.org",
+    email=getattr(settings, "API_ACCESS_FROM_EMAIL", None) or "api-requests@example.com"
```

Comment:
```suggestion
    email=email.API_ACCESS_FROM_EMAIL,
```

As I'm sure you're painfully aware, dynamic attribute lookup makes code harder to read and analyze, and Django already provides us a way to specify setting defaults. So, could you set `api-requests@example.com` as the default in cms/envs/common.py instead? 

If you'd rather not update edx-internal that's fine; studio.edx.org/api-docs will use `api-requests@example.com` either way.